src/components/app/app-theme/app.pallete.js

```diff
@@ -13,6 +13,7 @@ const blue = '#1976D2';
 const red = '#F44336';
 const lightPing = '#FEEFEF';
 const green = '#4CAF50';
+const shadow = 'inset 0 0 0 1000px rgba(0,0,0,.5)';
```

Comment:
Shadow is not applicable for current criteria.

Reply:
fixed